README.md

```diff
@@ -2866,6 +2874,126 @@ keyword 'default'.
 Virtual ip configuration. Valid values are an array of Strings or 'default'.
 
 --
+### Type: cisco_object_group

```

Comment:
So.. silly question.  Do we even need a `cisco_object_group` type?  What is the point of having one without an entry under the object group?